extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -110,6 +130,40 @@ export class LocalSubscriptionPlatform {
     return this.readerIdPromise_;
   }
 
+  /**
+   * Add event listener for the subscriptions action
+   * @private
+   */
+  initializeListeners_() {
+    this.document_.addEventListener('click', e => {
```

Comment:
Ditto. Only add listeners to `ampdoc.getRootNode()`

Reply:
Done